CHANGES.md

```diff
@@ -7,6 +7,7 @@
 ### Added
 
 + core: Add gnome_47 feature flag for GNOME 47
++ core: Add method `FactoryVecDeque::extend` to append multiple components efficiently.
```

Comment:
Sorry, it's sometimes not quite clear after a recent release where to put new items. This belongs below  `## Unreleased`. Just add a new `### Added` section there, otherwise it looks like this is part of a previous release.

Reply:
refactored